docs/samples/iot_hub_state_export.json

```diff
@@ -0,0 +1,554 @@
+{
```

Comment:
I bet a markdown table could help a user quickly gauge what is and isn't supported.